app/models/like.rb

```diff
@@ -1,2 +1,9 @@
 class Like < ApplicationRecord
+  belongs_to :user, foreign_key: 'author_id'
+  belongs_to :post, foreign_key: 'post_id'
+
+  def increment_like_counter
+    puts 'Incrementing like counter'
+    user.increment!(:likes_counter)
+  end
```

Comment:
- [ ] Please increment the likes_counter for a post, not the user as the likes should belong to a post, not a user.